### PR TITLE
Task/rnistuk/kep 1366

### PR DESCRIPTION
**Pull request by rnistuk**
Cleaned up the cmake files to ensure that the correct boost 1.70.0  include files are included, as opposed to those in the system folder. Also cleaned up a deprecated function warning.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -63,7 +55,7 @@ find_package(CURL REQUIRED)
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR})
 include_directories(SYSTEM ${OPENSSL_INCLUDE_DIR})
```
Comment (ebruck) [inline CMakeLists.txt]: remove these

File: CMakeLists.txt [hunk 2]
```diff
@@ -63,7 +55,7 @@ find_package(CURL REQUIRED)
...
-include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
+#include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
```
Comment (ebruck) [inline CMakeLists.txt]: remove this

Comment (rnistuk) [reply CMakeLists.txt]: Whups, missed this. Done

File: CMakeLists.txt (hunk 1 above)
Comment (rnistuk) [reply CMakeLists.txt]: Done

**Pull request merged**